Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -80,13 +80,14 @@ namespace {
     return properties;
   }
 
-  QWidget* table_view_builder(const std::shared_ptr<TableModel>& table, int row,
-      int column) {
+  QWidget* table_view_builder(
+      const std::shared_ptr<TimeAndSalesTableModel>& time_and_sales,
```

Comment:
What is the difference between ```time_and_sales``` and ```table```?

Reply:
`table` is the model of the `TableBody`, which is a `SortedTableModel`, while`time_and_sales` is the model of the `TableView`.

Refer to the process of creating `TableBody` model in the TableView:
```
 m_filtered_table = std::make_shared<FilteredTableModel>(
   m_table, std::bind_front(&TableView::is_filtered, this));
 if(comparator) {
   m_sorted_table = std::make_shared<SortedTableModel>(
     m_filtered_table, make_column_order(*m_header), std::move(comparator));
 } else {
   m_sorted_table = std::make_shared<SortedTableModel>(
     m_filtered_table, make_column_order(*m_header));
 }
 m_body = new TableBody(m_sorted_table, ...
```

Reply:
Thanks, that's something I should consider changing but for now your solution looks good.